contracts/test/EIP1271Wallet.vy

```diff
@@ -0,0 +1,78 @@
+# @version 0.3.3
+
+interface ERC20ApprovalInterface:
+    def approve(user: address, amount: uint256) -> bool: nonpayable
+
+interface NFTApprovalInterface:
+    def setApprovalForAll(user: address, approved: bool): nonpayable
+
+
+_EIP_1271_MAGIC_VALUE: constant(bytes4) = 0x1626ba7e
+
+owner: public(address)  # immutable
```

Comment:
idk how to declare public immutable

Reply:
Can't, but can make immutable constant and then reference in public view method